biolearn/data_library.py

```diff
@@ -256,6 +258,89 @@ def quality_report(self, sites=None):
 
         return QualityReport(sample_report, methylation_site_report, summary)
 
+    @staticmethod
+    def plot_sample_deviations(datasets, dataset_ids, output_path=None):
+        """
+        Creates a GeoData instance from a list of datasets and dataset ids
+        that can be used to visualize the distribution of sample deviations 
+        from the population mean in a ridge density plot.
+
+        Args:
+            datasets (list): A list of GeoData objects for each dataset.
+            dataset_ids (list): A list of dataset IDs (e.g., GSE IDs) for labeling.
```

Comment:
We should make the IDs available on the GeoData object as we discussed. That would make this argument unnecessary